static/usage/toast/styling/javascript.md

```diff
@@ -0,0 +1,37 @@
+```html
+<ion-button onclick="presentToast()">Click Me</ion-button>
+
+<script>
+  async function presentToast() {
+    const toast = document.createElement('ion-toast');
```

Comment:
While this works, we should be using the toast controller so the toast gets added as a child of `ion-app`.